inst/TMP/TEMP_generate_aux_data.R

```diff
@@ -59,3 +60,97 @@ fst::write_fst(tmp, paste0(data_folder_root, v1, "_aux/regions.fst"))
 # rgn_lkup <- fst::read_fst("TEMP/country_list.fst") %>%
 #   select(pcn_region_code, region = region) %>%
 #   distinct()
+
+
+# Country profile master --------------------------------------------------
+
+tmpfiles <- list.files(paste0(data_folder_root, v1, '/_aux'),
+                       pattern = 'indicator_values_country',
+                       full.names = TRUE)
+tmp <- lapply(tmpfiles, function(x) {
+  x <- read.csv(x)
+  x <- data.table::setDT(x)
+  names(x) <- tolower(sub('xyzD[MCP]xyz', '', names(x)))
+  x <- data.table::setnames(
+    x, skip_absent = TRUE,
+    c('country', 'requestyear', 'datayear', 'welfaretype',
+      'coverage', 'interpolation', 'survname', 'comparability',
+      'comparable_spell', 'povertyline', 'yearrange' #,
+      #'sp_pop_totl','si_pov_nahc', 'ny_gnp_pcap_cd', 'ny_gdp_mktp_kd_zg',
+      #'si_pov_gini', 'si_pov_theil'
```

Comment:
Hard coded indicator names will cause us trouble at some point... Can we pull them from the indicators master list instead?